packages/astro/src/i18n/index.ts

```diff
@@ -2,16 +2,31 @@ import { AstroError } from '../core/errors/index.js';
 import { MissingLocale } from '../core/errors/errors-data.js';
 import { shouldAppendForwardSlash } from '../core/build/util.js';
 import type { AstroConfig } from '../@types/astro.js';
-import { joinPaths } from '@astrojs/internal-helpers/path';
+import { appendForwardSlash, joinPaths } from '@astrojs/internal-helpers/path';
 
-type GetLocaleRelativeUrl = {
+type GetLocaleRelativeUrl = GetLocaleOptions & {
 	locale: string;
 	base: string;
 	locales: string[];
 	trailingSlash: AstroConfig['trailingSlash'];
 	format: AstroConfig['build']['format'];
 };
 
+export type GetLocaleOptions = {
+	/**
+	 * If the locale needs to be normalized
+	 */
```

Comment:
It would be nice to explain what "normalized" means in this context! Also, does this have a default value of `true` or `false`?